README.md

```diff
@@ -155,6 +155,8 @@ We use the [Development Project Board](https://github.com/getAlby/lightning-brow
 
 If you're a developer looking to help but not sure where to begin, check out the issues that have specifically been marked as being friendly to new contributors [here](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+project%3AgetAlby%2Flightning-browser-extension%2F2).
 
+> Alby enforces [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/) for adding human and machine readable meaning to commit messages, make sure to read and follow them.
```

Comment:
No need to link to a specific version I guess, just `https://www.conventionalcommits.org` is fine.

Reply:
Personally this would be enough for me but some projects explain the main things in a short summary. Let's see what the others think.

Reply:
There's a lot of stuff out there in the readme so kept it short. we can create a new markdown named Contributing Guidelines instead. WDYT?

Reply:
Good idea, but no need for now I think. Let's see what others think.  
These guides can be very helpful, i.e. https://github.com/LN-Zap/zap-desktop/blob/master/docs/CONTRIBUTING.md